src/mobile/src/libs/keychain.js

```diff
@@ -119,9 +121,43 @@ export const deleteTwoFactorAuthKeyFromKeychain = async () => {
 export const clearKeychain = async () => {
     await keychain.clear(ALIAS_SEEDS);
     await keychain.clear(ALIAS_AUTH);
+
     return await keychain.clear(ALIAS_SALT);
 };
 
+/**
+ * Gets realm encryption key.
+ *
+ * @method getRealmEncryptionKeyFromKeychain
+ * @returns {Promise}
+ */
+export const getRealmEncryptionKeyFromKeychain = () => {
+    return keychain.get(ALIAS_REALM).then((data) => {
+        const nonce = get(data, 'nonce');
+        const item = get(data, 'item');
+
+        if (nonce && item) {
+            return decodeBase64(item);
+        }
+
+        return null;
+    });
+};
+
+/**
+ * Sets realm encryption key.
+ *
+ * @method setRealmEncryptionKeyInKeychain
+ * @param {array} encryptionKey
+ *
+ * @returns {Promise}
+ */
+export const setRealmEncryptionKeyInKeychain = (encryptionKey) => {
+    return encodeBase64(encryptionKey).then((encodedEncryptionKey) =>
+        keychain.set(ALIAS_REALM, `Trinity-${getVersion()}`, encodedEncryptionKey),
```

Comment:
This key should probably be encrypted, because an application that spoofs the bundle ID could potentially read this value (this is only possible on a jailbroken device though)

Reply:
You are right. 

> I'm not sure what we would encrypt it with if we need it at load though
That's the problem. We don't have the password hash at this point. 

After having a discussion with @cvarley100 , we concluded that it's _okay_ to store it this way because we do warn users about the risks associated with using Trinity on a jailbroken/rooted device.

Reply:
Sounds good 👍